src/docs/components/methods.md

```diff
@@ -69,9 +69,15 @@ async myMethod3() {
   console.log(42);
 }
 
-// INVALID
+// INVALID: must use "async" keyword or return Promise
 @Method()
-notOk() {
+notOk() /* returnType: number */ {
+  return 42;
+}
+
+// ERROR: ts(1064) The return type of an async function or method must be the global Promise<T> type.
```

Comment:
This example seems off.

I don't think we should include this example because it's not stencil related, it's just how JS and TS. works. I would rather add a note about `async` functions, returning a Promise by spec, so the return type  is a Promise

Reply:
ok, looks like it was fixed, but still not sure about this example. It's not valid, not because of stencil, it's not valid, because of TS itself